joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/GradleDependencies.scala

```diff
@@ -227,7 +250,19 @@ object GradleDependencies {
                 Try(makeConnection(projectDir.toFile)) match {
                   case Success(connection) =>
                     Using.resource(connection) { c =>
-                      runGradleTask(c, initScript, initScriptFile.pathAsString)
+                      runGradleTask(c, initScript, initScriptFile.pathAsString) match {
+                        case Some(deps) =>
+                          val aarFileExtension = "aar"
```

Comment:
Maybe put all those special strings  
like aar or classes.jar in constants somewhere?

Reply:
good idea, done